test/unit/dom/childElementCount.js

```diff
@@ -38,67 +36,41 @@ describe('dom: childElementCount', () => {
       it('should return correct number of child nodes', () => {
         numbers.forEach(num => {
           expect(elem.childElementCount).to.equal(num);
-          if (type !== 'host' && canPatchNativeAccessors) {
```

Comment:
Most of the changes to the tests involve removing code like this. While it was definitely thorough to test, every single one of these would break Chrome because we're depending on double-underscored members to be available. I don't think the cost-benefit ratio here is worth testing these as we should be getting expected values by testing the public API.